.vscode/launch.json

```diff
@@ -5,10 +5,18 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Run Standalone (Chrome)",
+      "name": "Launch Standalone (Chrome)",
       "type": "chrome",
       "request": "launch",
-      "url": "http://localhost:3000",
+      "url": "http://localhost:3001/?server=localhost:8081&app=designer&pmv=variables&file=/variables.yaml",
```

Comment:
I think the file is useless as a parameter. This editor always works with the variables.yaml file. And there exists only one variables.yaml file per pmv...

I think it is ok to have this in the lsp protocol (as we maybe can share it for all config editors), but I would hard code it for this editor.

Or did you have a special thing in thought that you added it here?

Reply:
You are right, it is not necessary right now. I'm not sure how we are going to handle multiple config editors living here yet but what I was thinking is that once we have different editors we can identify the editor to use with this argument.
```
root.render(
  <React.StrictMode>
    <ThemeProvider defaultTheme={theme}>
      <ClientContextProvider client={client}>
        <QueryProvider client={queryClient}>
          {file === 'variables.yaml' && <VariableEditor app={app} pmv={pmv} file='variables.yaml' />}
          {file === 'roles.xml' && <RolesEditor app={app} pmv={pmv} file='roles.xml' />}
        </QueryProvider>
      </ClientContextProvider>
    </ThemeProvider>
  </React.StrictMode>
);
```
Then, we probably will have multiple launch configs with the different files in the url here. But I can remove it for now if you want me to.

Reply:
I would remove it for now, we can add it later again if we want to